control.html

```diff
@@ -209,16 +209,16 @@
     var key = evt.key || evt.which || evt.keyCode;
     var needUpdate = false;  // We'll use this to ignore key repeats
 
-    if (key === 38) {  // Up
+      if (key === "ArrowUp" || 38) {  // Up
```

Comment:
I think that the Javascript operator precedence will evaluate the === first, and then apply the || to the result, making the expression always evaluate to 38, and this conditional will always run. I think you want:

    if (key === "ArrowUp" || key === 38) {

If you want, I can make the change or you can do it and update the pull request.